### PR TITLE
Python basics updates

### PR DESCRIPTION
**Pull request by leemc-data-ed**
Minor updates, closes several issues:

#176: In the "Functions and methods" quiz on page 6 (line 227), the answer box discussed and answer that isn't one of the options. This has been fixed.

#195: Link to information about while loops was missing from page 11 (line 500); a resource has been added. 

#196: The code cells on page 15 had typos/errors in logic in the code that had them confusing. In the second cell, the third print statement is now print("This number is equal to or more than 100") (line 655). In the third cell, the code previously didn't account for all possibilities; new code has been added (lines 679-680).

### CODE REVIEW

File: python_basics_writing_python_code/python_basics_writing_python_code.md [hunk 1]
```diff
@@ -652,7 +652,7 @@ if num < 10:
 elif 10 < num < 100:
     print("This number is more than 10 but less than 100")
 elif num >= 100:
-    print("This number is more than 100")
+    print("This number is equal to or more than 100")
```
Comment (drelliche) [inline python_basics_writing_python_code/python_basics_writing_python_code.md:655]:
```suggestion
    print("This number is greater than or equal to 100")
```

Feel free to ignore this, I just made it match the other instance.

**Review by drelliche — COMMENTED** [review]
I found one typo and made one suggestion.

File: python_basics_writing_python_code/python_basics_writing_python_code.md [hunk 2]
```diff
@@ -224,7 +224,7 @@ Which of the following are valid examples of Python code? Select all that apply.
...
+These are all examples of valid Python code! `len("Python is awesome!")` returns the length of the argument `"Python is awesome!"` (which is 18-- spaces and punctuation count!) and prints that output to the screen. `"python.upper()"` makes the string "python" all uppercase. `type("4.3")` might be a little tricky, since 4.3 is a decimal number (or **float**), but it's also in quotation marks, which means that "4.3" is a string in this case! `type(True)` might also be tricky, since at first it looks like a string that doesn't have the required quotation marks. However, `True` and `False` are special in Python, and have the type `boolean`.
```
[the remark this answers is not in the document]
Comment (leemc-data-ed) [reply python_basics_writing_python_code/python_basics_writing_python_code.md:227]: Oooh good catch, TY!

**Comment by leemc-data-ed** [comment]
@drelliche thanks for taking a look at this! I've incorporated your suggested changes.

**Review by drelliche — APPROVED** [review]
lgtm!

**Pull request merged**